### PR TITLE
[ja] fix example ip address

### PR DESCRIPTION
**Pull request by pyama86**
I fixed easy miss.

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/nodelocaldns.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ NodeLocal DNSキャッシュは、クラスターノード上でDNSキャッシ
 {{< figure src="/images/docs/nodelocaldns.svg" alt="NodeLocal DNSCache flow" title="Nodelocal DNSCacheのフロー" caption="この図は、NodeLocal DNSキャッシュがDNSクエリーをどう扱うかを表したものです。" >}}
 
 ## 設定
-{{< note >}} NodeLocal DNSキャッシュ用のローカルに待ち受けているIPアドレスは、169.254.20.0/16の範囲のIPか、既存のIPと衝突しないことが保証されている他のIPとなります。このドキュメントでは例として169.254.10を使用します。
+{{< note >}} NodeLocal DNSキャッシュ用のローカルに待ち受けているIPアドレスは、169.254.20.0/16の範囲のIPか、既存のIPと衝突しないことが保証されている他のIPとなります。このドキュメントでは例として169.254.20.10を使用します。
```
Comment (ptux) [inline content/ja/docs/tasks/administer-cluster/nodelocaldns.md:48]: This jp block is out of date.
Would you translate the latest en block?

**Review by ptux — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/administer-cluster/nodelocaldns.md (hunk 1 above)
Comment (pyama86) [reply content/ja/docs/tasks/administer-cluster/nodelocaldns.md]: https://github.com/kubernetes/website/pull/31731/commits/bb948ff03fc8b682ad11a176c19af69aeefb1bc2
I have updated this block. please check.

File: content/ja/docs/tasks/administer-cluster/nodelocaldns.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ NodeLocal DNSキャッシュは、クラスターノード上でDNSキャッシ
 {{< figure src="/images/docs/nodelocaldns.svg" alt="NodeLocal DNSCache flow" title="Nodelocal DNSCacheのフロー" caption="この図は、NodeLocal DNSキャッシュがDNSクエリーをどう扱うかを表したものです。" >}}
 
 ## 設定
-{{< note >}} NodeLocal DNSキャッシュ用のローカルに待ち受けているIPアドレスは、169.254.20.0/16の範囲のIPか、既存のIPと衝突しないことが保証されている他のIPとなります。このドキュメントでは例として169.254.10を使用します。
+{{< note >}} NodeLocalDNSキャッシュのローカルリッスン用のIPアドレスは、クラスタ内の既存のIPと衝突しないことが保証できるものであれば、どのようなアドレスでもかまいません。例えば、IPv4のリンクローカル範囲169.254.0.0/16やIPv6のユニークローカルアドレス範囲fd00::/8から、ローカルスコープのアドレスを使用することが推奨されています。
```
Comment (ptux) [inline content/ja/docs/tasks/administer-cluster/nodelocaldns.md:48]:
```suggestion
{{< note >}} NodeLocal DNSキャッシュのローカルリッスン用のIPアドレスは、クラスタ内の既存のIPと衝突しないことが保証できるものであれば、どのようなアドレスでもかまいません。例えば、IPv4のリンクローカル範囲169.254.0.0/16やIPv6のユニークローカルアドレス範囲fd00::/8から、ローカルスコープのアドレスを使用することが推奨されています。
```

**Review by ptux — CHANGES_REQUESTED** [review]

**Comment by pyama86** [comment]
thanks.
What should i do for merge?

**Comment by nasa9084** [comment]
Thank you for your contribution!
Please create an issue at first, and (in the next time) please assign an approver as k8s-ci-robot commented like:
/assign @nasa9084

**Comment by pyama86** [comment]
@nasa9084 
Hi!

> Please create an issue at first, and (in the next time) please assign an approver as k8s-ci-robot commented like:

ok, I do next time!
BTW, It seems to some label need to add in this PR, would you please add it?

**Pull request merged**